cmd/operator/internal/controller/common.go

```diff
@@ -0,0 +1,346 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package controller
+
+// a13e => shorthand for ApiServerSource
+// k9e  => shorthand for KubeArchive
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"sort"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	sourcesv1 "knative.dev/eventing/pkg/apis/sources/v1"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	meta "k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	schema "k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	kubearchivev1alpha1 "github.com/kubearchive/kubearchive/cmd/operator/api/v1alpha1"
+)
+
+const (
+	SinkFilterConfigMapName   = "sink-filters"
+	ApiServerSourceLabelName  = "kubearchive.org/enabled"
+	ApiServerSourceLabelValue = "true"
+	GlobalNamespace           = "___global___"
```

Comment:
If you have the sink import this variable maybe it doesn't make sense for it to be in `cmd/operator/internal` since it would become shared

Reply:
at the same time, the SinkFiltersConfigMapName could also be imported by the sink to remove duplication of constants as well

Reply:
Done.